src/cdk/scrolling/scrolling-module.ts

```diff
@@ -14,8 +15,9 @@ import {CdkVirtualForOf} from './virtual-for-of';
 import {CdkVirtualScrollViewport} from './virtual-scroll-viewport';
 
 @NgModule({
-  imports: [PlatformModule],
+  imports: [BidiModule, PlatformModule],
```

Comment:
I don't think you need to import the `BidiModule` since the `Directionality` is provided at the root.

Reply:
I think it definitely needs to be in `exports`, so that we detect native `dir="rtl"` attrs as directives. I'm not sure if it needs to be in `imports`, I see a lot of other modules doing it though. @jelbourn do you know if having it in `imports` is still necessary?